--- dump-0.4b43/configure.in.orig	2011-02-03 19:33:01.000000000 +0100
+++ dump-0.4b43/configure.in	2011-02-03 19:37:25.852692227 +0100
@@ -510,7 +510,7 @@
 dnl
 dnl Check for rl_completion_matches
 dnl
-AC_CHECK_LIB(readline, rl_completion_matches, [rlcm=yes], [rlcm=no], "-ltermcap")
+AC_CHECK_LIB(readline, rl_completion_matches, [rlcm=yes], [rlcm=no], "$rdllib")
 if test "$rlcm" = yes; then
 	AC_DEFINE([HAVE_READLINE_RLCM],1,[Define this if your readline libs have the rl_completion_matches library.])
 fi
@@ -518,7 +518,7 @@
 dnl
 dnl Check for rl_completion_append_character
 dnl
-AC_CHECK_LIB(readline, rl_completion_append_character, [rcac=yes], [rcac=no], "-ltermcap")
+AC_CHECK_LIB(readline, rl_completion_append_character, [rcac=yes], [rcac=no], "$rdllib")
 if test "$rcac" = yes; then
 	AC_DEFINE([HAVE_READLINE_CAC],1,[Define this if your readline libs have the rl_completion_append_character variable.])
 fi
