--- dump-0.4b33/dump/Makefile.in.orig	Tue Feb 11 23:18:12 2003
+++ dump-0.4b33/dump/Makefile.in	Tue Feb 11 23:19:54 2003
@@ -13,12 +13,10 @@
 
 PROG=		dump
 RPROG=		rdump
-LINKS=		${SBINDIR}/dump ${SBINDIR}/rdump
 SRCS=		itime.c main.c optr.c tape.c traverse.c unctime.c
 OBJS=		itime.o main.o optr.o tape.o traverse.o unctime.o \
 		../common/dumprmt.o
-MAN8=		dump.8
-RMAN8=		rdump.8
+MAN8=		dump.8 rdump.8
 
 all::		$(PROG) $(MAN8)
 
@@ -31,11 +29,11 @@
 	    -e "s|__VERSION__|$(VERSION)|g" $< > $@
 
 install::	all
-	$(INSTALL) -d $(SBINDIR) $(MANDIR)
-	$(INSTALLBIN) $(PROG) $(SBINDIR)
-	$(INSTALLMAN) $(MAN8) $(MANDIR)
-	cd $(SBINDIR) && $(RM) -f $(RPROG) && $(LN_S) $(PROG) $(RPROG)
-	cd $(MANDIR) && $(RM) -f $(RMAN8) && $(LN_S) $(MAN8) $(RMAN8)
+	$(INSTALL) -d $(DESTDIR)$(sbindir) $(DESTDIR)$(mandir)
+	$(INSTALLBIN) $(PROG) $(DESTDIR)$(sbindir)
+	$(LN_S) dump $(DESTDIR)$(sbindir)/rdump
+	$(INSTALLMAN) $(MAN8) $(DESTDIR)$(mandir)
+
 
 clean::
 	$(RM) -f $(PROG) $(MAN8) \#* *.s *.o *.a *~ core
--- dump-0.4b33/rmt/Makefile.in.orig	Tue Feb 11 23:22:03 2003
+++ dump-0.4b33/rmt/Makefile.in	Tue Feb 11 23:23:07 2003
@@ -3,6 +3,8 @@
 top_srcdir=	@top_srcdir@
 srcdir=		@srcdir@
 top_builddir=	..
+sbindir=	@sbindir@
+mandir=		@mandir@
 
 @MCONFIG@
 
@@ -26,9 +28,10 @@
 	    -e "s|__VERSION__|$(VERSION)|g" $< > $@
 
 install::	all
-	$(INSTALL) -d $(SBINDIR) $(MANDIR)
-	$(INSTALLBIN) $(PROG) $(SBINDIR)
-	$(INSTALLMAN) $(srcdir)/$(MAN8) $(MANDIR)
+	$(INSTALL) -d $(DESTDIR)$(sbindir) $(DESTDIR)$(mandir)
+	$(INSTALLBIN) $(PROG) $(DESTDIR)$(sbindir)
+	$(INSTALLMAN) $(MAN8) $(DESTDIR)$(mandir)
+	
 
 clean::
 	rm -f $(PROG) \#* *.s *.o *.a *~ core rmt.8
--- dump-0.4b33/MCONFIG.in.orig	Tue Feb 11 23:11:18 2003
+++ dump-0.4b33/MCONFIG.in	Tue Feb 11 23:11:43 2003
@@ -22,8 +22,8 @@
 
 prefix=		@prefix@
 exec_prefix=	@exec_prefix@
-SBINDIR=	@sbindir@
-MANDIR=		@mandir@/man8
+sbindir=	@sbindir@
+mandir=		@mandir@/man8
 
 DUMPDATESPATH=	@DUMPDATESPATH@
 #
--- dump-0.4b33/Makefile.in.orig	Tue Feb 11 23:14:05 2003
+++ dump-0.4b33/Makefile.in	Tue Feb 11 23:14:42 2003
@@ -9,7 +9,7 @@
 
 all clean install dep depend realclean distclean::
 	for i in $(SUBDIRS); do \
-		(cd $$i && $(MAKE) $@) || exit 1; \
+		(cd $$i && $(MAKE) DESTDIR=$(DESTDIR) $@) || exit 1; \
 	done
 
 distclean::
--- dump-0.4b35/configure.in.orig	2003-04-10 08:25:21.000000000 +0000
+++ dump-0.4b35/configure.in	2003-11-23 12:46:11.000000000 +0000
@@ -498,12 +498,6 @@
 AC_CHECK_TYPE(u_quad_t, uint64_t)
 
 dnl
-dnl Compute top_buildir
-dnl
-top_builddir=`cd .; pwd`
-AC_SUBST(top_builddir)
-
-dnl
 dnl Create directories
 dnl
 test -d compat || mkdir compat
@@ -512,4 +507,5 @@
 dnl
 dnl Output files
 dnl
-AC_OUTPUT(MCONFIG Makefile common/Makefile compat/include/Makefile compat/lib/Makefile dump/Makefile restore/Makefile $RMTMAKEFILE)
+AC_CONFIG_FILES([MCONFIG Makefile common/Makefile compat/include/Makefile compat/lib/Makefile dump/Makefile restore/Makefile $RMTMAKEFILE])
+AC_OUTPUT
--- dump-0.4b33.orig/restore/rrestore.8 Thu Jan  1 01:00:00 1970
+++ dump-0.4b33/restore/rrestore.8      Mon Nov 18 15:51:55 2002
@@ -0,0 +1 @@
+.so restore.8
--- dump-0.4b32.orig/dump/rdump.8       Thu Jan  1 01:00:00 1970
+++ dump-0.4b32/dump/rdump.8    Mon Nov 18 15:51:55 2002
@@ -0,0 +1 @@
+.so dump.8
--- dump-0.4b40/restore/Makefile.in.orig	2005-05-02 17:10:46.000000000 +0200
+++ dump-0.4b40/restore/Makefile.in	2005-06-14 17:33:13.056818552 +0200
@@ -19,8 +19,7 @@
 		utilities.c xattr.c
 OBJS=		dirs.o interactive.o main.o restore.o symtab.o tape.o \
 		utilities.o xattr.o ../common/dumprmt.o
-MAN8=		restore.8
-RMAN8=		rrestore.8
+MAN8=		restore.8 rrestore.8
 
 .c.o:
 	$(CC) -c $(ALL_CFLAGS) $< -o $@
@@ -35,11 +34,11 @@
 	    -e "s|__VERSION__|$(VERSION)|g" $< > $@
 
 install::	all
-	$(INSTALL) -d $(SBINDIR) $(MANDIR)
-	$(INSTALLBIN) $(PROG) $(SBINDIR)
-	$(INSTALLMAN) $(MAN8) $(MANDIR)
-	cd $(SBINDIR) && $(RM) -f $(RPROG) && $(LN_S) $(PROG) $(RPROG)
-	cd $(MANDIR) && $(RM) -f $(RMAN8) && $(LN_S) $(MAN8) $(RMAN8)
+	$(INSTALL) -d $(DESTDIR)$(sbindir) $(DESTDIR)$(mandir)
+	$(INSTALLBIN) $(PROG) $(DESTDIR)$(sbindir)
+	$(LN_S) restore $(DESTDIR)$(sbindir)/rrestore
+	$(INSTALLMAN) $(MAN8) $(DESTDIR)$(mandir)
+
 
 clean::
 	$(RM) -f $(PROG) $(MAN8) \#* *.s *.o *.a *~ core
