diff -Nru dump-0.4b22/MCONFIG.in dump-0.4b22.new/MCONFIG.in
--- dump-0.4b22/MCONFIG.in	Sat May 12 13:39:45 2001
+++ dump-0.4b22.new/MCONFIG.in	Sun May 13 02:09:29 2001
@@ -26,8 +26,8 @@
 
 prefix=		@prefix@
 exec_prefix=	@exec_prefix@
-SBINDIR=	@sbindir@
-MANDIR=		@mandir@/man8
+sbindir=	@sbindir@
+mandir=		@mandir@
 
 DUMPDATESPATH=	@DUMPDATESPATH@
 #
diff -Nru dump-0.4b22/dump/Makefile.in dump-0.4b22.new/dump/Makefile.in
--- dump-0.4b22/dump/Makefile.in	Thu Apr 12 18:03:29 2001
+++ dump-0.4b22.new/dump/Makefile.in	Sun May 13 02:22:22 2001
@@ -12,12 +12,10 @@
 
 PROG=		dump
 RPROG=		rdump
-LINKS=		${SBINDIR}/dump ${SBINDIR}/rdump
 SRCS=		itime.c main.c optr.c tape.c traverse.c unctime.c
 OBJS=		itime.o main.o optr.o tape.o traverse.o unctime.o \
 		../common/dumprmt.o
-MAN8=		dump.8
-RMAN8=		rdump.8
+MAN8=		dump.8 rdump.8
 
 all::		$(PROG) $(MAN8)
 
@@ -30,11 +28,10 @@
 	    -e "s|__VERSION__|$(VERSION)|g" $< > $@
 
 install::	all
-	$(INSTALL) -d $(SBINDIR) $(MANDIR)
-	$(INSTALLBIN) $(PROG) $(SBINDIR)
-	$(INSTALLMAN) $(srcdir)/$(MAN8) $(MANDIR)
-	cd $(SBINDIR) && $(RM) -f $(RPROG) && $(LN_S) $(PROG) $(RPROG)
-	cd $(MANDIR) && $(RM) -f $(RMAN8) && $(LN_S) $(MAN8) $(RMAN8)
+	$(INSTALL) -d $(DESTDIR)$(sbindir) $(DESTDIR)$(mandir)/man8
+	$(INSTALLBIN) $(PROG) $(DESTDIR)$(sbindir)
+	$(LN_S) dump $(DESTDIR)$(sbindir)/rdump
+	$(INSTALLMAN) $(MAN8) $(DESTDIR)$(mandir)/man8
 
 clean::
 	$(RM) -f $(PROG) $(MAN8) \#* *.s *.o *.a *~ core
diff -Nru dump-0.4b22/dump/rdump.8 dump-0.4b22.new/dump/rdump.8
--- dump-0.4b22/dump/rdump.8	Thu Jan  1 01:00:00 1970
+++ dump-0.4b22.new/dump/rdump.8	Sun May 13 02:08:51 2001
@@ -0,0 +1 @@
+.so dump.8
diff -Nru dump-0.4b22/restore/Makefile.in dump-0.4b22.new/restore/Makefile.in
--- dump-0.4b22/restore/Makefile.in	Thu Apr 12 18:03:30 2001
+++ dump-0.4b22.new/restore/Makefile.in	Sun May 13 02:18:31 2001
@@ -2,6 +2,8 @@
 
 top_srcdir=	@top_srcdir@
 srcdir=		@srcdir@
+mandir=		@mandir@
+sbindir=	@sbindir@
 
 @MCONFIG@
 
@@ -17,8 +19,7 @@
 		utilities.c
 OBJS=		dirs.o interactive.o main.o restore.o symtab.o tape.o \
 		utilities.o ../common/dumprmt.o
-MAN8=		restore.8
-RMAN8=		rrestore.8
+MAN8=		restore.8 rrestore.8
 
 all::		$(PROG) $(MAN8)
 
@@ -30,11 +31,9 @@
 	    -e "s|__VERSION__|$(VERSION)|g" $< > $@
 
 install::	all
-	$(INSTALL) -d $(SBINDIR) $(MANDIR)
-	$(INSTALLBIN) $(PROG) $(SBINDIR)
-	$(INSTALLMAN) $(srcdir)/$(MAN8) $(MANDIR)
-	cd $(SBINDIR) && $(RM) -f $(RPROG) && $(LN_S) $(PROG) $(RPROG)
-	cd $(MANDIR) && $(RM) -f $(RMAN8) && $(LN_S) $(MAN8) $(RMAN8)
+	$(INSTALL) -d $(DESTDIR)$(sbindir) $(DESTDIR)$(mandir)/man8
+	$(INSTALLBIN) $(PROG) $(DESTDIR)$(sbindir)
+	$(INSTALLMAN) $(srcdir)/$(MAN8) $(DESTDIR)$(mandir)/man8
 
 clean::
 	$(RM) -f $(PROG) $(MAN8) \#* *.s *.o *.a *~ core
diff -Nru dump-0.4b22/restore/rrestore.8 dump-0.4b22.new/restore/rrestore.8
--- dump-0.4b22/restore/rrestore.8	Thu Jan  1 01:00:00 1970
+++ dump-0.4b22.new/restore/rrestore.8	Sun May 13 02:08:51 2001
@@ -0,0 +1 @@
+.so restore.8
diff -Nru dump-0.4b22/rmt/Makefile.in dump-0.4b22.new/rmt/Makefile.in
--- dump-0.4b22/rmt/Makefile.in	Fri Apr  6 12:09:40 2001
+++ dump-0.4b22.new/rmt/Makefile.in	Sun May 13 02:20:27 2001
@@ -2,6 +2,8 @@
 
 top_srcdir=	@top_srcdir@
 srcdir=		@srcdir@
+sbindir=	@sbindir@
+mandir=		@mandir@
 
 @MCONFIG@
 
@@ -30,9 +32,9 @@
 	    -e "s|__VERSION__|$(VERSION)|g" $< > $@
 
 install::	all
-	$(INSTALL) -d $(SBINDIR) $(MANDIR)
-	$(INSTALLBIN) $(PROG) $(SBINDIR)
-	$(INSTALLMAN) $(srcdir)/$(MAN8) $(MANDIR)
+	$(INSTALL) -d $(DESTDIR)$(sbindir) $(DESTDIR)$(mandir)/man8
+	$(INSTALLBIN) $(PROG) $(DESTDIR)$(sbindir)
+	$(INSTALLMAN) $(MAN8) $(DESTDIR)$(mandir)/man8
 
 clean::
 	rm -f $(PROG) \#* *.s *.o *.a *~ core rmt.8
--- dump-0.4b22/Makefile.in~	Tue Nov  2 10:35:55 1999
+++ dump-0.4b22/Makefile.in	Sun May 13 02:25:34 2001
@@ -8,7 +8,7 @@
 
 all clean install dep depend realclean distclean::
 	for i in $(SUBDIRS); do \
-		(cd $$i && $(MAKE) $@) || exit 1; \
+		(cd $$i && $(MAKE) DESTDIR=$(DESTDIR) $@) || exit 1; \
 	done
 
 distclean::
