--- dump-0.4b35/configure.in.orig	2003-12-23 08:16:56.949563872 +0100
+++ dump-0.4b35/configure.in	2003-12-23 08:18:17.914255368 +0100
@@ -143,12 +143,12 @@
 	READLINE=""
 	echo "Not including readline support"
 else
-	READLINE="-lreadline -ltermcap"
+	READLINE="-lreadline -lncurses"
 	AC_DEFINE([HAVE_READLINE],1,[Define if you want to include readline support.])
 	echo "Including readline support"
 fi
 ,
-READLINE="-lreadline -ltermcap"
+READLINE="-lreadline -lncurses"
 AC_DEFINE([HAVE_READLINE],1,[Define if you want to include readline support.])
 echo "Including readline support by default"
 )
