--- dump-0.4b36/dump/main.c.orig	2004-04-21 10:55:51.000000000 +0200
+++ dump-0.4b36/dump/main.c	2004-06-19 00:32:33.747372855 +0200
@@ -66,6 +66,8 @@
 #include <ext2fs/ext2fs.h>
 #include <sys/stat.h>
 #include <bsdcompat.h>
+/* <linux/audit.h> includes <asm/types.h> which conflict with <ext2fs/ext2_fs.h> on some archs */
+#define _LINUX_AUDIT_H_
 #include <linux/fs.h>	/* for definition of BLKFLSBUF */
 #elif defined sunos
 #include <sys/vnode.h>
